### PR TITLE
Doc Updates for 1.0

### PR DESCRIPTION
**Pull request by cwanda**
1) Updated Check stats on a BLE device tutorial
   -  Removed versions and updated paths.
   - Rename BLE Device to NRF52 Board
   - Added build step for myble target
   - Used new b scan parameters
2) Add load bootloader section to BLE HCI Tutorial
3) Use "main" task instead of "default" task in Add Console and Shell to Blinky tutorial.
4) Remove what the cputime module might be used for
5) Updated Console doc:
   - Added description for minimal package
   - Added newline parameter to console_read function
6) Concepts section - removed "features", added aflags and lflags
7) Use 1-latest instead of 0-latest in Newt Tool Intro
8) Creating first project tutorial: fixed test "libs/os" package to "sys/config" package
   and fixed paths listed in debugger out
9) Changed required GO version to 1.7
10)  Added intro page for blinky tutorials and moved common stuff (description, prerequisites) from individual blinky tutorials to the intro page.
11) Added intro page for slinky tutorials and moved common stuff (description, prerequisites) from
   individual slinky tutorials to the intro page.
12) Renamed project-target-slinky to project-nrf52-slinky and updated link references

### CODE REVIEW

**Comment by utzig** [comment]
Let us know when this is OK for review/merge.

**Comment by cwanda** [comment]
Hi Fabio,
These are ready for review. 
Thanks,
Wanda

File: docs/os/tutorials/blinky.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+###Overview of Steps
+These are the general steps to create, load and run the Blinky application on your board:
+
+* Create a project.
+* Define the bootloader and Blinky application targets for the board.
+* Build the bootloader target.
+* Build the Blinky application target and create an application image.
+* Connect to the board.
+* Load to bootloader onto the board.
+* Load to Blinky application image onto the board.
```
Comment (aditihilbert) [inline docs/os/tutorials/blinky.md]: will change "to" to "the" before merge

File: docs/os/tutorials/blinky.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+* See the LED on your board blink.
+
+<br>
+
+After you try the Blinky application on your boards, checkout out other tutorials to enable additional functionality such as [remote comms](project-slinky.md) on the current board.
+
```
Comment (aditihilbert) [inline docs/os/tutorials/blinky.md]: will add check stat project too

File: docs/os/tutorials/project-slinky.md [hunk 3]
```diff
@@ -1,142 +1,36 @@
...
 * Have Internet connectivity to fetch remote Mynewt components.
-* Have a computer to build a Mynewt application.
+* Have a computer to build a Mynewt application and connect to the board over USB.
+* Have a Micro-USB cable to connect the board and the computer.
+* Have a [serial port setup](/os/get_started/serial_access.md).
 * Install the newt tool and the toolchains (See [Basic Setup](/os/get_started/get_started.md)).
 * Install the [newtmgr tool](../../newtmgr/installing/).
+* Create a project space (directory structure) and populated it with the core code repository (apache-mynewt-core) or kn
+ow how to as explained in [Creating Your First Project](/os/get_started/project_create).
 * Read the Mynewt OS [Concepts](/os/get_started/vocabulary.md) section.
```
Comment (aditihilbert) [inline docs/os/tutorials/project-slinky.md]: will move this up

File: mkdocs.yml [hunk 4]
```diff
@@ -47,10 +48,11 @@ pages:
...
         - 'Events and Event Queues': 'os/tutorials/event_queue.md'
-        - 'Project Slinky for remote comms':
-            - 'Slinky on sim device': 'os/tutorials/project-slinky.md'
-            - 'Slinky on Nordic nRF52 board': 'os/tutorials/project-target-slinky.md'
-            - 'Slinky on STM32 board': 'os/tutorials/project-stm32-slinky.md'
+        - 'Project Slinky for Remote Comms':
+            - toc: 'os/tutorials/project-slinky.md'
+            - 'Slinky on sim device': 'os/tutorials/project-sim-slinky.md'
+            - 'Slinky on Nordic nRF52': 'os/tutorials/project-nrf52-slinky.md'
+            - 'Slinky on Olimex': 'os/tutorials/project-stm32-slinky.md'
```
Comment (aditihilbert) [inline mkdocs.yml]: I will reorder these a little: move up Slinky and Checks stats in the list so people can go down the list in increasing order of complexity/new concepts.

**Comment by aditihilbert** [comment]
Not sure why it "closed" it instead of "merging". But I merged all the changes.

**Pull request closed without merging**